tests/filtertransformers/test_elasticsearch.py

```diff
@@ -6,7 +6,9 @@
 )
 
 from optimade.filterparser import LarkParser
-from optimade.filtertransformers.elasticsearch import ElasticTransformer, Quantity
+from optimade.filtertransformers.elasticsearch import (
+    ElasticTransformer,
+)
```

Comment:
```suggestion
from optimade.filtertransformers.elasticsearch import ElasticTransformer
```

Reply:
Don't know if this is a line length issue, perhaps?

Reply:
hehe, just tried this...

```
$ echo "from optimade.filtertransformers.elasticsearch import ElasticTransformer" > test.py; black test.py; cat test.py
All done! ✨ 🍰 ✨
1 file left unchanged.
from optimade.filtertransformers.elasticsearch import ElasticTransformer
```

and
```
$ echo "from optimade.filtertransformers.elasticsearch import (\nElasticTransformer\n)" > test.py; black test.py; cat test.py
reformatted test.py
All done! ✨ 🍰 ✨
1 file reformatted.
from optimade.filtertransformers.elasticsearch import ElasticTransformer
```

BUT (note the comma)

```
$ echo "from optimade.filtertransformers.elasticsearch import (\nElasticTransformer,\n)" > test.py; black test.py; cat test.py
reformatted test.py
All done! ✨ 🍰 ✨
1 file reformatted.
from optimade.filtertransformers.elasticsearch import (
    ElasticTransformer,
)
```

Using this as a justification to ignore your suggestion :grin: